gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -827,14 +828,14 @@ private static void checkContentType(BlobInfo blobInfo) throws IllegalArgumentEx
    *
    * @throws StorageException upon failure
    */
-  ListResult<BucketInfo> list(BucketListOption... options);
+  Page<BucketInfo> list(BucketListOption... options);
```

Comment:
Unrelated to this PR. Can we fix `StorageExample` to iterate over the complete result set?

To avoid this kind of oversight do you think we should not make Page Iterable but rather have `values` method instead?